src/components/UndoTools.tsx

```diff
@@ -0,0 +1,29 @@
+import { useIntl } from "react-intl";
+import { UndoToolsObject } from "./singleton-editor";
+
+export function UndoTools({ canUndo, canRedo, undo, redo }: UndoToolsObject) {
+  const intl = useIntl();
+  const undoText = intl.formatMessage({ id: "undo" });
+  const redoText = intl.formatMessage({ id: "redo" });
+
+  return (
+    <div className="undo-redo--container">
+      <button
+        className="icon-undo"
+        onClick={undo}
+        title={undoText}
+        disabled={!canUndo}
+      >
+        {undoText}
+      </button>
+      <button
+        className="icon-redo"
+        onClick={redo}
+        title={redoText}
+        disabled={!canRedo}
+      >
+        {redoText}
+      </button>
```

Comment:
@GusBaamonde, please help me to define this HTML structure and classes. 

I will also need the changes in the Style Guide.

![image](https://github.com/FromDoppler/doppler-editors-webapp/assets/1157864/05676a0f-3df0-4f4b-87b0-547b40bcb138)